styles.css

```diff
@@ -16,3 +16,15 @@
   @import "react-responsive-carousel/lib/styles/carousel.min.css";
 }
 
+.breadcrumb_item {
+    @apply inline text-[0.8rem] text-[#666];
+}
+
+.breadcrumb_item a {
+    @apply no-underline;
+}
```

Comment:
Move these our of styles.css. Use Tailwind classes as much as possible.

Reply:
Yeah these are specific to breadcrumbs only, can we pull the styles next to the component (even better if it was in the component)